src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -4,10 +4,23 @@ import PropTypes from 'prop-types';
 import { getExamsBaseUrl } from '../data/api';
 
 const getLaunchUrlByExamId = (id) => `${getExamsBaseUrl()}/lti/exam/${id}/instructor_tool`;
+const ltiToolEmbed = false;
+
+function openNewTab(url) {
+  window.open(url, '_blank');
+}
 
 const ExternalReviewDashboard = ({ exam }) => (
+  // By default, launch instructor tool in a new tab.
+  // Otherwise, embed in dashboard.
   <div data-testid="review_dash">
-    {exam && <iframe title="lti_tool" src={getLaunchUrlByExamId(exam.id)} width="100%" height="1100" style={{ border: 'none' }} />}
+    <div>
+      {!ltiToolEmbed && exam && openNewTab(getLaunchUrlByExamId(exam.id))}
```

Comment:
This is going to call window.open on every render not just when the tab is clicked.